yhondanilo/chess.rb

```diff
@@ -0,0 +1,48 @@
+class Chess
+  attr_accessor :fila, :columna
+
+  def initialize(fila, columna)
+    @fila = fila
+    @columna = columna
+    intercalar
+  end
+  def intercalar
+    @board=Array.new(@fila){Array.new(@columna)}
+    @board[0][0]=1
+    for f in 0..self.fila-1
+      for c in 0..self.columna-1
+        if @board[f][c-1]==0
+          @board[f][c]=1
+        elsif @board[f][c-1]==1
+          @board[f][c]=0
+        end
+        #if @board[f][c]==@board[f][@columna-1]
+        if c==@columna-1
+          ultimo=@board[f][c]
+          if ultimo==0
+            @board[f+1][c]=1
+          elsif ultimo==1
+            @board[f+1][c]=0
+          end
+        end
+      end
+    end
+  end
+  def imprimir
+    for f in 0..self.fila-1
+      for c in 0..self.columna-1
```

Comment:
Prefer `each` over `for`.<br>Redundant `self` detected.<br>Surrounding space missing for operator '-'.